app/services/rad_common/search.rb

```diff
@@ -192,5 +192,11 @@ def active_users
       def inactive_users
         Pundit.policy_scope!(current_user, User).inactive.by_name
       end
+
+      def enum_scopes(model, enum)
```

Comment:
this is awesome, probably should have a maint update to refactor existing projects